source/_integrations/zwave_me.markdown

```diff
@@ -38,6 +38,28 @@ This integration allows you to control a Z-Wave network via the [Z-Wave.Me Z-Way
 
 {% include integrations/config_flow.md %}
 
+In the URL field input the IP address with the port. The IP address can be prefixed with wss:// if HTTPS should be used instead of HTTP (when using find.z-wave.me remote access or public IP with SSL).
```

Comment:
Please use `{% configuration_basic %}` for describing config flow fields.

Reply:
Should we just add it before line 41?

Reply:
> ```shell
> git checkout my_feature
> git rebase --onto next my_feature~3
> ```

Sure about --onto next ?
And 3 is the number of commits my_featuer is ahead of next?